ORK1Kit/ORK1Kit/Common/ORK1TaskViewController.m

```diff
@@ -966,11 +976,14 @@ - (void)showViewController:(ORK1StepViewController *)viewController goForward:(B
     // from the same VC.
     _currentStepViewController = viewController;
     
-    NSString *progressLabel = nil;
-    if ([self shouldDisplayProgressLabel]) {
+    ORK1TaskProgress taskProgress;
+    taskProgress.current = 0;
+    taskProgress.total =  0;
```

Comment:
These 3 lines could be shortened to `ORK1TaskProgress taskProgress = ORK1TaskProgressMake(0, 0);`